packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionIcon.ts

```diff
@@ -6,13 +6,15 @@ export const getActionIcon = (status: TaskStatus | 'CANCELED'): IconVariant => {
     case 'PENDING': {
       return 'action-progress';
     }
-    case 'COMPLETE':
-    case 'OVERWRITE_PREVENTED': {
+    case 'COMPLETE': {
       return 'action-success';
     }
     case 'FAILED': {
       return 'action-error';
     }
+    case 'OVERWRITE_PREVENTED': {
+      return 'info';
```

Comment:
So these icons names stay aligned let's add an 'action-info` icon variant that uses the underlying 'info' icon

Reply:
Updated